### PR TITLE
[fix](functions) add quantile_state_empty function signature

### PR DESCRIPTION
**Pull request by zhangstar333**
## Proposed changes
quantile_state_empty = percentile_empty
to_quantile_state        = percentile_hash 
quantile_percent        = percentile_approx_raw

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zhangstar333** [comment]
run buildall

**Review by HappenLee — COMMENTED** [review]
Add nereids function and add test case

**Comment by zhangstar333** [comment]
run buildall

**Comment by zhangstar333** [comment]
run buildall

File: be/src/vec/functions/function_quantile_state.cpp [hunk 1]
```diff
@@ -191,14 +191,28 @@ class FunctionQuantileStatePercent : public IFunction {
         }
         auto str_col = assert_cast<const ColumnQuantileState*>(column.get());
         auto& col_data = str_col->get_data();
-        auto percent_arg = check_and_get_column_const<ColumnFloat32>(
-                block.get_by_position(arguments.back()).column);
+        float percent_arg_value = -1.0;
+        const auto& [percent_column, percent_column_const] =
+                unpack_if_const(block.get_by_position(arguments[1]).column);
 
-        if (!percent_arg) {
```
Comment (HappenLee) [inline be/src/vec/functions/function_quantile_state.cpp:197]: why remove the logic，the second arg should be const

**Comment by zhangstar333** [comment]
run buildall

**Comment by zhangstar333** [comment]
run buildall

**Review by HappenLee — APPROVED** [review]
LGTM

**Review by Gabriel39 — APPROVED** [review]
LGTM

**Pull request merged**